app/src/componentLibrary/autocompletes/multipleAutocomplete/multipleDownshift.jsx

```diff
@@ -26,9 +26,10 @@ export const MultipleDownshift = ({
   onChange,
   handleUnStoredItemCb,
   options,
+  isExistingLaunchNames,
   ...props
 }) => {
-  const [storedItemsMap, setStoredItems] = useState({});
+  const [storedItemsMap, setStoredItems] = useState(isExistingLaunchNames);
```

Comment:
I thought about that, when implement this task.
But  I decided to did not change anything to not brake anything. I will try to simplify it.

Reply:
You're right. Now this is not necessary and may be better added to the technical debt. The recipient field also uses this component, so it's better to leave it as is
But I suggest renaming the isExistingLaunchNames property to AsyncMultipleAutocomplete and MultipleDropdown to something more generic since it's a common component